src/_utils.ts

```diff
@@ -158,3 +158,19 @@ export function doesDependencyExist(
     return false;
   }
 }
+
+export function sanitizePackageManagerName(name: string): {
+  name: any;
+  warning: string | undefined;
+} {
+  const sanitized = /\w+$/.exec(name)?.[0];
```

Comment:
Even though today we don't have any package manager with `-` in it, it could be possible.